tests/fixtures/dist/drupal-conventions.yml

```diff
@@ -0,0 +1,6 @@
+imports:
```

Comment:
To be more explicit, we could name this file drupal-conventions-with-exclude.

Reply:
Maybe in future we can use it for other things drupal related.

Reply:
ok